<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="pipline_test.MyUtilsTest" duration-ms="65" started-at="2019-04-19T09:36:53Z" finished-at="2019-04-19T09:36:54Z">
    <groups>
    </groups>
    <test name="Command line test" duration-ms="65" started-at="2019-04-19T09:36:53Z" finished-at="2019-04-19T09:36:54Z">
      <class name="pipline_test.MyUtilsTest">
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:pipline_test.MyUtilsTest@2d209079]" name="beforeClass" is-config="true" duration-ms="9" started-at="2019-04-19T17:36:53Z" finished-at="2019-04-19T17:36:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="testAdd()[pri:0, instance:pipline_test.MyUtilsTest@2d209079]" name="testAdd" duration-ms="4" started-at="2019-04-19T17:36:54Z" finished-at="2019-04-19T17:36:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testAdd -->
        <test-method status="FAIL" signature="testMul()[pri:0, instance:pipline_test.MyUtilsTest@2d209079]" name="testMul" duration-ms="0" started-at="2019-04-19T17:36:54Z" finished-at="2019-04-19T17:36:54Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<2.0> but was:<1.0>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<2.0> but was:<1.0>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:553)
	at org.junit.Assert.assertEquals(Assert.java:683)
	at pipline_test.MyUtilsTest.testMul(MyUtilsTest.java:38)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMul -->
        <test-method status="PASS" signature="testSub()[pri:0, instance:pipline_test.MyUtilsTest@2d209079]" name="testSub" duration-ms="0" started-at="2019-04-19T17:36:54Z" finished-at="2019-04-19T17:36:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSub -->
        <test-method status="PASS" signature="afterClass()[pri:0, instance:pipline_test.MyUtilsTest@2d209079]" name="afterClass" is-config="true" duration-ms="1" started-at="2019-04-19T17:36:54Z" finished-at="2019-04-19T17:36:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- pipline_test.MyUtilsTest -->
    </test> <!-- Command line test -->
  </suite> <!-- pipline_test.MyUtilsTest -->
</testng-results>
